fluent-dom/src/localization.js

```diff
@@ -55,17 +37,55 @@ export default class Localization {
    */
   async formatWithFallback(keys, method) {
     const translations = [];
+    let hasMissing = true;
+    let debugInfo = null;
+
     for (let ctx of this.ctxs) {
       // This can operate on synchronous and asynchronous
       // contexts coming from the iterator.
       if (typeof ctx.then === "function") {
         ctx = await ctx;
       }
-      const errors = keysFromContext(method, ctx, keys, translations);
-      if (!errors) {
+      const missingIds = keysFromContext(method, ctx, keys, translations);
+      if (missingIds === null) {
+        hasMissing = false;
         break;
+      } else {
+        if (debugInfo === null) {
+          debugInfo = new Map();
+        }
+        debugInfo.set(
+          ctx.locales[0],
+          missingIds
+        );
+      }
+    }
+
+    if (hasMissing) {
```

Comment:
By _remove `hasMissing`_ I meant removing this block. We still need the variable to break out of the loop, of course.